pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -199,6 +215,17 @@ func New(
 
 	ctrl.featureGateAccess = featureGateAccess
 
+	ctrl.configInformerFactory = configInformerFactory
+
+	enabled, err := ctrl.sigstoreAPIEnabled()
+	if enabled {
+		if err := ctrl.addImagePolicyObservers(); err != nil {
+			klog.Infof("skipping error adding image policy observers: %v", err)
+		}
+	} else {
+		klog.Infof("skipping adding image policy observers with SigstoreImageVerification Featuregate value: %v, error: %v", enabled, err)
+	}
```

Comment:
Do we even need to try adding the observers already in `New`? Is there any downside to doing it only in `Run`, and saving some of the bookkeeping complexity? (Maybe some tests need it to happen here?)

Reply:
Can rebase and test, I think tests should be ok to simplify this method call and logs.